.github/CONTRIBUTING.md

```diff
@@ -35,3 +35,13 @@ your contributions.
   on save ([see VS Code one here](https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode)).
 - We have a git hook to automatically run prettier before commit, in case you don't install the plugin.
 - If for any reason, prettier does not work for you, you can run `yarn format:prettier` and that should format everything.
+- When using VS Code, configuration files have been provided to take advantage of their [container development infrastructure](https://code.visualstudio.com/docs/remote/containers). touted benefits:
+
+    > The Remote - Containers extension lets you use a Docker container as a full-featured development environment. Whether you deploy to containers or not, containers make a great development environment because you can:
+    >
+    > - Develop with a consistent, easily reproducible toolchain on the same operating system you deploy to.
+    > - Quickly swap between different, separate development environments and safely make updates without worrying about impacting your local machine.
+    > - Make it easy for new team members / contributors to get up and running in a consistent development environment.
+    > - Try out new technologies or clone a copy of a code base without impacting your local setup.
+  
+  To get started add the [Remote Containers](https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.remote-containers) extension and follow installation instructions provided there.
```

Comment:
Get rid of all but the first sentence.
> When using VS Code, configuration files have been provided to take advantage of their [container development infrastructure](https://code.visualstudio.com/docs/remote/containers).

Is fine, the rest doesn't belong in our readme.

Reply:
addressed in: https://github.com/thelounge/thelounge/pull/4514/commits/30c50438f4ea7b2045d30418a5ce10aed9ff8721